azafea/event_processors/endless/metrics/v2/migrations/d4809698244a_add_duration.py

```diff
@@ -0,0 +1,28 @@
+# type: ignore
+
+"""Add duration column to open shell apps
+
+Revision ID: d4809698244a
+Revises: 4a49897b47ea
+Create Date: 2020-04-30 13:54:30.888222
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd4809698244a'
+down_revision = '4a49897b47ea'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('shell_app_is_open', sa.Column('duration', sa.Float(), nullable=True))
+    op.execute('UPDATE shell_app_is_open SET duration = 0')
+    op.alter_column('shell_app_is_open', 'duration', nullable=False)
```

Comment:
@adarnimrod does your last comment mean you think this should be split into different migrations?

Reply:
I meant that we can have `null` values in the interim before updating all of the records and afterwards change it so that `null` aren't accepted (this change will be in a separate PR). But I don't think that's necessary (0 or -1 is fine by me).